.github/workflows/ci.yml

```
@@ -119,6 +135,62 @@ jobs:
         flags: unittests
         name: codecov-umbrella
 
+  unit-test-modules:
+    name: Unit Tests (Modules)
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: crypto
+            make1: no-op
```

I think it's clearer if we can see the setup phase on the CI level:
```suggestion
make1: -C crypto setup
```

That way we can also delete the new job added under `crypto/Makefile` (`test_main`).

---

We could even call `make1` as `setup` , and `make2` as `test` for clarity.

---

Yeah, good idea. I updated it in 639938f7074ad10068f847dcec8830060fcb546f